tests/test_asttokens.py

```diff
@@ -137,5 +137,24 @@ def test_coding_declaration(self):
       "ENDMARKER:''"
     ])
 
+
+def test_filename():
+  filename = "myfile.py"
+  atok = asttokens.ASTTokens("a", parse=True, filename=filename)
+  assert filename == atok.filename
+
+
+def test_doesnt_have_location():
```

Comment:
Could you add a comment what this tests, please? The test name isn't self-explanatory.